File: SecondYear/adityshrma08_AdityaSharma_2226csit1014_2/Week_2/Count Digits.cpp
int countDigits(int n){
	// Write your code here.
	int r,count=0,t;
	t = n;
	while(t!=0){
		r = t%10;
		if(r>0){
		if(n%r==0){
			count ++;
		}
		}
		t = t/10;
	}
		return count;
}

File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_2/CountDigits.java
public class CountDigits {
        public static int countDigits(int n) {
            // Write your code here.
            int temp = n;
            int count = 0;
            int digit;
            while (temp > 0) {
                digit = temp % 10;
                if (digit != 0) {
                    if (n % digit == 0) {
                        count++;
                    }
                }
                temp = temp / 10;
            }
            return count;
        }
}
